addons/l10n_rs_edi/wizard/account_move_send.py

```diff
@@ -0,0 +1,106 @@
+from odoo import _, api, fields, models, SUPERUSER_ID
+from odoo.exceptions import ValidationError
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = 'account.move.send'
+
+    l10n_rs_edi_send_enable = fields.Boolean(compute="_compute_l10n_rs_edi_send_enable")
+    l10n_rs_edi_send_readonly = fields.Boolean(compute="_compute_l10n_rs_edi_send_readonly")
+    l10n_rs_edi_send_checkbox = fields.Boolean(
+        string="eFaktura",
+        compute="_compute_l10n_rs_edi_send_checkbox",
+        store=True, readonly=False,
+        help="Send the E-Invoice to Government via eFaktura"
+        )
+    l10n_rs_edi_send_cir_checkbox = fields.Boolean(
+        string="Send To CIR",
+        default=False,
+        help="Send to Central Invoice Register(For B2G and the public sector)"
+    )
+
+    @api.depends('move_ids.l10n_rs_edi_state')
```

Comment:
```suggestion
    @api.depends('move_ids')
```
`move_ids` is not supposed to change. By removing this part from the dependency, you prevent the orm to make a new query everytime `l10n_rs_edi_state` on any invoice